README.md

```diff
@@ -347,9 +378,77 @@ knows Staltz is part of the group because all `group/add-member` messages have
 Staltz can see Arj has invited Mix because he's replicating Arj's
 "group/additions" feed, so Staltz starts replicating Mix's group feed.
 
+## 5. Security Considerations
+
+### 5.1. Degraded confidentiality at scale
+
+As noted in the [private-groups-original-notes], SSB Private Groups are based on
+symmetric encryption and static keys (no key rotation), which means it cannot
+guarantee [perfect-forward-secrecy] neither [post-compromise-security]. This
+makes SSB Private Groups less likely to guarantee confidentiality at scale, when
+a group has dozens or hundreds of members. If any of the members leaks the group
+secret, all past communication in the group is compromised.
+
+This also means that any group member can add new members to the group. They may
+even do so without publishing a `group/add-member` message, which effectively
+allows any group member to invite a third party that will remain undetected by
+existing group members.
+
+SSB Private Groups with several members should thus not be considered
+confidential nor forward secret, and should be used with caution. Our security
+model assumes that the group is as trustworthy as the _least_ trustworthy member
+in the group.
+
+The choice of symmetric encryption (as opposed to, e.g. hash-ratchet or
+double-ratchet) was made because of efficiency. Symmetric encryption means
+adding new members or publishing new content is `O(1)` fast, and the system is
+overall simple to implement.
+
+### 5.2. Membership analysis attacks
+
+A group feed is discoverable only by the group members, because its announcement
+is encrypted. However, an eavesdropper who can replicate a peer's metafeed tree
+will notice the presence of these encrypted message on shard feeds. If the
+eavesdropper can do that for several peers, then they may perform a metadata
+analysis attack where they try to correlate the encrypted messages on the shard
+feeds to the group members.
+
```

Comment:
this will need weaving in a little more, but I think we should mention this, it's a security feature

We can also say timing attacks could be possible, but because there are unknown delays in the network, and in when people "accept" and set up a group feed, it makes it harder.
We could also further mitigate by only setting up the group feed when you go to publish in the group for the first time - adds even more delay.